portfolio/src/main/webapp/script.js

```diff
@@ -15,34 +15,58 @@
 // limitations under the License.
  
 /**
- * Adds a random greeting to the page.
+ * Gives the user a random book recommendation.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
- 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
- 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+
  
-function sayRandomQuote(){
-    const quotes = [
-        "Elementary, my dear Watson",
-        "You miss 100% of the shots you don't take." ,
-        "The greatest glory in living lies not in never falling, but in rising every time we fall.",
-        "The future belongs to those who believe in the beauty of their dreams.",
-        "Do not go where the path may lead, go instead where there is no path and leave a trail." ,
-        "Life is really simple, but we insist on making it complicated." ,
+function sayRandomBook(){
+    const books = [
+        "Jane Eyre", "1984","Don Quixote", "The Great Gatsby", "Harry Potter and the Philosopher's Stone",
+        "War and Peace", "The Alchemist", "The Lord of the Rings", "Things fall apart", "Lord of the Flies", "Animal Farm",
+        "The Lion, The Witch, and the Wardrobe","Percy Jackson and the Lightning Thief", "Honor among thieves", "Summer Nights at the Midnight Hotel", "It", "The Hound of the Baskervilles",
     ];
  
-    const chosenQuote= quotes[Math.floor(Math.random() * quotes.length)];
+    const chosenbook= books[Math.floor(Math.random() * books.length)];
  
-    const quoteContainer = document.getElementById("quote-container");
-    quoteContainer.innerHTML = chosenQuote;
+    const bookContainer = document.getElementById("book-container");
+    bookContainer.innerHTML = chosenbook;
  
 }
- 
+
+/**
+ * Adds a new project name, info and picture everytime the equivalent button is clicked.
+ */
+
+let project = 0;
+const projects = ["This was my first ever python project. I used file handling and some basic data structures to store information from users who create accounts and then that information can later be accessed in order to gain entry into the system. Priority was also added to each user, on account creation, depending on the password strength so users with weaker passwords had access to less sensitive information.",
```

Comment:
There's a lot of written content here - IMO (though I'm not a front-end expert), I think it would be better to put all of the content in the HTML and have only the code in the javascript. That way, if you wanted to edit the content of the page, then you would only need to edit the html file.

You could have all three projects in the HTML, but selectively show/hide the divs of each project using the javascript.
https://www.w3schools.com/howto/howto_js_toggle_hide_show.asp